lib/stylesheet/manager.rb

```diff
@@ -271,7 +299,24 @@ def plugins_digest
   end
 
   def settings_digest
-    Digest::SHA1.hexdigest((theme&.included_settings || {}).to_json)
+    timestamps = ""
+    proc = Proc.new do |&blck|
```

Comment:
I don't feel like the code here justifies the use of a `Proc`

```
timestamps = ThemeField.where("(name = ? AND type_id = ? AND theme_id = ?) OR (theme_id = ?) ORDER BY updated_at ASC").pluck(:updated_at).map!(&:to_f).join(",")
```

Perhaps something like the above?